docs/manage-metadata-using-gravitino.md

```diff
@@ -820,6 +820,7 @@ The following types that Gravitino supports:
 | Map                       | `Types.MapType.of(keyType, valueType)`                                   | `{"type": "map", "keyType": type JSON, "valueType": type JSON, "valueContainsNull": JSON Boolean}`                                   | Map type, indicate a map of key-value pairs                                                      |
 | Struct                    | `Types.StructType.of([Types.StructType.Field.of(name, type, nullable)])` | `{"type": "struct", "fields": [JSON StructField, {"name": string, "type": type JSON, "nullable": JSON Boolean, "comment": string}]}` | Struct type, indicate a struct of fields                                                         |
 | Union                     | `Types.UnionType.of([type1, type2, ...])`                                | `{"type": "union", "types": [type JSON, ...]}`                                                                                       | Union type, indicates a union of types                                                           |
+| Unparsed(unparsedType)    | `Types.UnparsedType.of(unparsedType)`                                    | `{"type": "unparsed", "unparsedType": "user-defined"}`                                                                               | Unparsed type, indicates an unresolvable type                                                    |
```

Comment:
I believe it would be beneficial to introduce a new five-level heading to describe the `unparsedType`, similar to [unparsedExpression](https://datastrato.ai/docs/0.4.0/expression#unparsed-expression). This way, we can simply refer to it in other instances where the `unparsedType` needs to be described, consolidating the description of its purpose.

Reply:
Additionally, this will allow for clear differentiation between `unparsedType` and other types, as it is only used for `loadTable` and not for `createTable`.